### PR TITLE
[Submission] Steinmetz

### PR DESCRIPTION
**Pull request by csteinmetz1**
**Steinmetz**

Dear mdx workshop committee,

I request a review for the following MDX submission:

### **Deep learning for automatic mixing: challenges and next steps**  

**Author(s): Christian J. Steinmetz**  

**Category ***

* [ ] `POSTERS` _(+Abstract: 250 Words+one page pdf)_
  > As a participant of the MDX challenge, please know that your poster would be automatically accepted after a minimal prescreening. Hence, don't forget to provide us with the link to your team in the AIcrowd website.
  > You are also welcome to submit us some unrelated abstract, in case your interest would be in presenting some method or idea you would just like to discuss with the participants passing by.  
  > => You *don't* need to include your poster in the PR. Just be ready to have it for D-day and show it to the audience through screen sharing.
* [x] `LONG TALK`
  > A long talk will last about 20-30min, including questions. This can be about some recent paper you published, or may also present some original research and investigations you did. If you pick this category, please provide us with an extended abstract of the talk, or directly with the slides if you have them available, even in an unfinished state, at your convenience. 
* [ ] `DISCUSSIONS`
  > This category is intended to stimulate fruitful discussions and collaborations. If you pick it, it means you are willing to chair some open discussion with the participants, mainly around one particular aspect of music demixing research. For that purpose, you should provide a minimal abstract as well as a list of a few questions you would like the participants to discuss about. Don't hesitate to propose ideas, We will possibly contact you in case we identify several similar topics that were proposed!

**Preferred Timezone***

* [x] A 11:00 PM (PDT) to 9:00 AM PDT
* [ ] B 11:00 AM (PDT) to 9:00 PM PDT

  > The preferred timezone will help us to organize the day schedule. You should select at least one, but can perfectly select both if you don't care. We will organize the programme so that to maximally fit your preferences.
---

**ORGANISATION COMMITTEE (do not fill out)**

* [x] Editor acknowledgment
* [x] Reviewer 1 
* [x] Reviewer 2
* [x] Review 1 decision [accept]
* [ ] Review 2 decision [accept/reject]
* [ ] Editor decision [accept/reject]

### CODE REVIEW

**Comment by aliutkus** [comment]
Hi, thanks a lot for the submission
is there somewhere we could see some extended version of the content you intend to present ?

**Comment by csteinmetz1** [comment]
Some of the content will be shared from a talk I gave with JT Colonel at an AES Convention, which you can see [here](https://www.youtube.com/watch?v=Jb34r4CAJGE). I will cover also a few newer papers and also talk about some relevant papers on source separation for mixing, which wasn't mentioned in the previous talk.

**Review by aliutkus — COMMENTED** [review]
review for one key application of source separation research, which is automatic remixing.
I'm curious whether you address the multichannel scenario from mono or degraded mixtures, that is definitely a very desired feature in real applications

File: paper.md [hunk 1]
```diff
@@ -1,139 +1,24 @@
...
-
-For a quick reference, the following citation commands can be used:
-- `@author:2001`  ->  "Author et al. (2001)"
-- `[@author:2001]` -> "(Author et al., 2001)"
-- `[@author1:2001; @author2:2001]` -> "(Author1 et al., 2001; Author2 et al., 2002)"
+The process of transforming a set of audio recordings into a cohesive mixture for film, radio, or music production encompasses a number of artistic and technical considerations. 
+Due to the inherent complexity in this task, a great deal of training and expertise on the part of the audio engineer is generally required. 
+This has motivated the design of automated or assistive systems for the audio production process that aim to aid both amateur and expert users. 
+While classical machine learning approaches and expert systems have been investigated, they often fail to generalize to the diversity and scale of real-world projects, with the inability to adapt to a varying number of sources, capture stylistic elements across genres, or apply sophisticated processing. 
+Recently, deep learning approaches have shown promise in addressing these limitations and bring the potential to model the complex mixing process directly from data.
```
Comment (aliutkus) [inline paper.md:22]: I think it would be good to include references

**Review by aliutkus — COMMENTED** [review]
It's ok to keep the abstract short, but it would be good to give a few pointers. If you published some overview paper about this topic recently, it'd be a good idea to include a reference

**Review by faroit — APPROVED** [review]
I have nothing further to add. As mentioned by @aliutkus I suggest to add a few references even when all from yourself.

**Comment by faroit** [comment]
@csteinmetz1 see our reviews above

**Comment by aliutkus** [comment]
To all participants:
please note that you should update your submission by November 6th 23:00 UTC

Thank you very much

**Comment by csteinmetz1** [comment]
Thanks for the comments. I have updated the abstract with some relevant references.

**Comment by faroit** [comment]
👏 The submission has been published on midspace (the ISMIR conference platform). The submissions will also be hosted on the workshop website tomorrow. 

The schedule can be seen there or [here](https://docs.google.com/spreadsheets/d/1PaksFcHWUfNPMg-XloA3CJtrvb9uMU0erCFVyvFi9B4/edit?usp=sharing). Please let us know if there is anything missing. Otherwise we see each other on Friday/Saturday!

**Comment by faroit** [comment]
Hi @csteinmetz1 , sorry for the long waiting time but I wasn't able to continue working on the proceedings before. 

I managed to finally work on cutting the video recordings of the workshop and plan to upload the recordings to YouTube very soon. In order to allow me to do this, please acknowledge the following:  

> I hereby authorize the right and permission to copyright and/or publish, reproduce or otherwise use my name, voice, and audio-visual recordings. I acknowledge and understand these materials about or of me may be used for both commercial and/or non-commercial purposes.

> I understand that my image may be edited, copied, exhibited, published and/or distributed. There is no time limit on the validity of this release nor are there any geographic limitations on where these materials may be distributed.

> I authorize that the video will be made available under the [CC BY-SA 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/) on the conference website.

and comment on this by replying with `acknowledge`.

**Comment by csteinmetz1** [comment]
`acknowledge`

Thanks @faroit!

**Pull request closed without merging**